typescript/packages/subsurface-viewer/src/SubsurfaceViewer.tsx

```diff
@@ -41,15 +42,15 @@ export type LightsType = {
     pointLights?: [
         {
             intensity: number;
-            position: [number, number, number];
+            position: Point3D;
```

Comment:
I wonder if the changed types in the API is causing `dash-generate-components` to fail? Have a look at `SubsurfaceViewer.propTypes`, maybe it needs to be declared?

Reply:
I'm afraid we'd still like to have `propTypes` due to it checking prop types during runtime and our props coming in from Dash/Python via JSON.